src/msw/webview_ie.cpp

```diff
@@ -857,6 +857,67 @@ wxString wxWebViewIE::GetPageText() const
     }
 }
 
+wxWebViewIEEmulationLevel wxWebViewIE::GetEmulationLevel()
+{
+    wxRegKey key(wxRegKey::HKCU, _T("SOFTWARE\\Microsoft\\Internet Explorer\\Main\\FeatureControl\\FEATURE_BROWSER_EMULATION"));
+    if ( key.Exists() )
+    {
+        long val = 0;
+
+        if ( !key.QueryValue(wxGetFullModuleName().AfterLast('\\'), &val) )
+        {
+            wxLogWarning(_("Can not get level value of registry key \
+                SOFTWARE\\Microsoft\\Internet Explorer\\Main\\FeatureControl\\FEATURE_BROWSER_EMULATION"));
+            return wxWEBVIEWIE_EMULATION_LEVEL_EMPTY;
+        }
+
+        switch ( val )
+        {
+            case wxWEBVIEWIE_EMULATION_LEVEL_11:
+                return wxWEBVIEWIE_EMULATION_LEVEL_11;
+            case wxWEBVIEWIE_EMULATION_LEVEL_11_DEFAULT:
+                return wxWEBVIEWIE_EMULATION_LEVEL_11_DEFAULT;
+            case wxWEBVIEWIE_EMULATION_LEVEL_10:
+                return wxWEBVIEWIE_EMULATION_LEVEL_10;
+            case wxWEBVIEWIE_EMULATION_LEVEL_10_DEFAULT:
+                return wxWEBVIEWIE_EMULATION_LEVEL_10_DEFAULT;
+            case wxWEBVIEWIE_EMULATION_LEVEL_9:
+                return wxWEBVIEWIE_EMULATION_LEVEL_9;
+            case wxWEBVIEWIE_EMULATION_LEVEL_9_DEFAULT:
+                return wxWEBVIEWIE_EMULATION_LEVEL_9_DEFAULT;
+            case wxWEBVIEWIE_EMULATION_LEVEL_8:
+                return wxWEBVIEWIE_EMULATION_LEVEL_8;
+            case wxWEBVIEWIE_EMULATION_LEVEL_8_DEFAULT:
+                return wxWEBVIEWIE_EMULATION_LEVEL_8_DEFAULT;
+            case wxWEBVIEWIE_EMULATION_LEVEL_7_DEFAULT:
+                return wxWEBVIEWIE_EMULATION_LEVEL_7_DEFAULT;
+            default:
+                return wxWEBVIEWIE_EMULATION_LEVEL_EMPTY;
```

Comment:
It is not correct that `wxFAIL_MSG` is only available in debug mode, but I don't think it makes sense here neither, it's not a programming error if this code runs on a newer version of Windows with some new emulation level defined.

Just return the same `UNKNOWN` level as mentioned above.